### DISCUSSION TITLE
Wie findet ihr die Funktionen?

### DISCUSSION THREAD

**Discussion by samueltrippler** [malangmann/GitHub-Aufgabe_Gruppe1#4] [General]
Wie sinnvoll findet ihr die Funktionen, die bereits hochgeladen sind?
Bei 3 d) soll man eine metrische und eine dichotome Variable in Beziehung setzen. Da kommen doch nur Alter und Mathe-LK in Frage, oder?
Bei 3 c) haben wir gedacht, dass man hier Studienfach und Mathe-LK vergleichen soll. In der Aufgabe steht nur "zwei kategoriale Variablen", aber wenn man jetzt "Interesse an ..." nehmen würde, dann hätte man, ohne das vorher nochmals zu kategorisieren, extrem viele Merkmalsausprägungen, die man miteinander vergleichen müsste.

**Comment by tmmmarius** [comment]
Bei der 3b) haben wir eine Funktion, welche die relative Häufigkeitstabelle ausgibt. Fallen euch andere deskriptive Statistiken ein, welche hier passen könnten oder ist die Tabelle ausreichend?

**Comment by malangmann** [comment]
Korrelationskoeffizienzen könnten noch hilfreich sein. 
Falls deskriptive Statistiken für metrische Variablen (3a)) für die Auswertung fehlen, füge ich die gerne hinzu.

**Reply by samueltrippler** [reply]
Worauf bezieht sich das genau? Für die Korrelationskoeffizienten müssten wir zwei Variablen in Beziehung setzen, am besten zwei metrische. Da haben wir nur das Alter. Oder an welche Variablen hast du genau gedacht?

**Comment by samueltrippler** [comment]
Mir ist aufgefallen, dass wir noch die Studienfächer ins Verhältnis zu den Interessen setzen müssen, damit wir Zusammenhänge erkennen können, deshalb habe ich noch eine Funktion hinzugefügt.

Ehrlich gesagt bin ich ein bisschen unsicher, wie wir hier die zu Grunde liegenden Zusammenhänge überhaupt erkennen können ...
Brauchen wir vielleicht noch weitere Funktionen, damit gewisse Zusammenhänge deutlicher werden? ^^
